src/main/java/frc/robot/Constants.java

```diff
@@ -5,132 +5,141 @@
+    public static final int FR = 12;
+    public static final int BL = 13;
+    public static final int BR = 14;
+    public static final int BIG_ARM = 6;
+    public static final int BIG_ARM_2 = 7;
+    public static final int SMALL_ARM = 31;
+    public static final int Compressor = 1;
+  }
+
+  public static final class BigArmPosition {
+    public static final double HIGH = 0.578857;
+    public static final double MEDIUM = 0.437500;
+    public static final double PICKUP = 0.392578;
+    public static final double DEFAULT = 0.386475;
+    public static final double SLIDER = 0.595092; // Fine tune
+    public static final double PICKUPCONE = 0.47295;
+  }
+
+  public static final class isInverted {
+    // This is for moters
+    public static final boolean FL = false;
+    public static final boolean FR = true;
+    public static final boolean BL = false;
+    public static final boolean BR = true;
+  }
+
+  public static final class Canbus {
+    public static final String CANBUS = "rio";
+  }
+
+  public static final class Canivore {
+    public static final String CANIVORE = "drivetrain";
+  }
+
+  public static final class Sensor {
+    public static final String CANIVORE = "drivetrain";
+    public static final int CUBE_CHANNEL = 2;
+    public static final int CONE_CHANNEL = 3;
+  }
+
+  public static final class Gamepad {
+    public static final int DRIVE_CONTROLLER = 0;
+    public static final int ARM_CONTROLLER = 1;
+
+    public static final int LEFT_JOYSTICK_XAXIS = 0;
+    public static final int LEFT_JOYSTICK_YAXIS = 1;
+    public static final int LEFT_TRIGGER = 2;
+    public static final int RIGHT_TRIGGER = 3;
+    public static final int RIGHT_JOYSTICK_XAXIS = 4;
+
+    public static final int A_BUTTON = 1;
+    public static final int B_BUTTON = 2;
+    public static final int X_BUTTON = 3;
+    public static final int Y_BUTTON = 4;
+    public static final int LEFT_BUTTON = 5;
+    public static final int RIGHT_BUTTON = 6;
+    public static final int BACK_BUTTON = 7;
+    public static final int START_BUTTON = 8;
+    public static final int L_JOYSTICK_BUTTON = 9;
+    public static final int R_JOYSTICK_BUTTON = 10;
+  }
+
+  public static final class WheelOffset {
+    // Converting rotations to radians
+    public static final double FRONT_LEFT_ENCODER = (0.793945) * 2 * Math.PI;
+    public static final double FRONT_RIGHT_ENCODER = (0.489502) * 2 * Math.PI;
+    public static final double BACK_LEFT_ENCODER = (0.234375) * 2 * Math.PI;
+    public static final double BACK_RIGHT_ENCODER = (0.520752) * 2 * Math.PI;
+  }
+
+  public static final class MotorGearRatio {
+    public static final double DRIVE = 8.14; // this is the ratio of bot
+    public static final double ANGLE =
+        12.8; // https://www.swervedrivespecialties.com/products/mk4-swerve-module
+    public static final int BIG = 10;
+    public static final int SMALL = 7;
+  }
+
+  public static final class SmallArmPosition {
```

Comment:
Nested classes again for these `Constants.Positions.Small.HIGH` and `Constants.Positions.Big.PICKUP` etc